generate_test.go

```diff
@@ -51,9 +51,8 @@ message test {
   repeated ufixed64 sux64 = 108 [packed=true];
   repeated float sf32 = 109 [packed=true];
   repeated double sf64 = 110 [packed=true];
-  repeated bytes sbytes = 111;
```

Comment:
This is a *slice* of *slice* of bytes. This is exactly what we don't want to encode anymore.